include/configs/imx8mm_var_dart.h

```diff
@@ -88,9 +88,9 @@
 	"swu=1\0" \
 	"get_fit_config=leica_mcu_comm board_id board_id;" \
 		"if test \"${board_id}\" < \"D\"; then" \
-		"    env set fit_config conf-freescale_ap20-pt1.dtb;" \
+		"    DISPLAY ERROR MESSAGE AND NO BOOT" \
```

Comment:
NAK: This is neither a _U-Boot_ command nor meaningful message. Since this gets executed - it **will** lead to an error, so it has to be re-worked.

Reply:
it my fault, i have updated to `echo !!! No support board_id:\"{board_id}\", boot failed;`, and i have test on my evk board, it can show msg in the serial port. thank you.